docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/06-Send Transaction.mdx

```diff
@@ -370,3 +285,18 @@ Finally, the transaction is signed, encoded, and submitted to the network. A cli
     ```
   </TabItem>
 </Tabs>
+
+## Handling transactions with Wallets in web apps.
+#### Creating Noop Signers
+When sending transactions from your web application, users need to sign the transaction using their wallet. Since the transaction needs to assembled beforehand, you can create a `noopSigner` (no-operation signer). This will act as a placeholder for you instructions, indicating that a given account is a signer and the signature wil be added later. After the user of your web app signs the transaction with their wallet, you need to manually add that signature to the transaction.
```

Comment:
Think you can just pass the serialized transaction to the wallet extension and they will return a serialized transaction (with the added signature). `add the signature manually` might sound a little more complicated than it actually is